packages/app-content-pages/next.config.js

```diff
@@ -34,13 +34,21 @@ const nextConfig = {
   assetPrefix,
   basePath: '/about', 
 
+  compiler: {
+    styledComponents: true,
+  },
+
   env: {
     COMMIT_ID,
     PANOPTES_ENV,
     SENTRY_CONTENT_DSN,
     APP_ENV
   },
 
+  experimental: {
+    forceSwcTransforms: true,
```

Comment:
I'm having trouble finding the related documentation in the next.js docs?
The styledComponents docs are here - https://nextjs.org/docs/advanced-features/compiler#styled-components .

Reply:
It’s linked from the SWC warning in the build log. I don’t know if it’s documented anywhere else. 

https://nextjs.org/docs/messages/swc-disabled

Reply:
Also, seems a bit harsh to turn off SWC if you use Babel to run tests, like we do.